packages/providers/src/utils/change-case/functions.ts

```diff
@@ -0,0 +1,298 @@
+// Regexps involved with splitting words in various case formats.
+const SPLIT_LOWER_UPPER_RE = /([\p{Ll}\d])(\p{Lu})/gu;
+const SPLIT_UPPER_UPPER_RE = /(\p{Lu})([\p{Lu}][\p{Ll}])/gu;
```

Comment:
It looks like the names of these got clipped, let's add back the end for consistency.

```suggestion
const SPLIT_LOWER_UPPER_REGEX = /([\p{Ll}\d])(\p{Lu})/gu;
const SPLIT_UPPER_UPPER_REGEX = /(\p{Lu})([\p{Lu}][\p{Ll}])/gu;
```